libkpod/remove.go

```diff
@@ -16,8 +16,11 @@ func (c *ContainerServer) Remove(container string, force bool) (string, error) {
 	}
 	ctrID := ctr.ID()
 
-	cState := c.runtime.ContainerStatus(ctr)
-	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
+	cStatus := c.runtime.ContainerStatus(ctr)
+	switch cStatus.Status {
+	case oci.ContainerStatePaused:
```

Comment:
In testing, it looked like I'd some kind of timing error here if I allowed the force to remove the pause container.  Sometimes it worked, sometimes it didn't.  I'd say about 75% work.  Need to play more.  To be safe I've removed the --force logic for now.